packages/gatsby-source-wordpress/src/normalize.js

```diff
@@ -608,3 +608,16 @@ exports.createNodesFromEntities = ({ entities, createNode }) => {
     })
   })
 }
+
+exports.createUrlPathsFromLinks = entities =>
+  entities.map(e => {
+    if (e.link && !e.path) {
+      try {
+        const link = new URL(e.link)
```

Comment:
I had some trouble with Netlify refusing the build because `URL` is not imported in the file. I am not sure if this is required here for production builds. Maybe a Node version change is required – Netlify uses version 8 by default. Switching to version 10 of Node solved most of the past issues i had.

Reply:
Yeah, I think we need to import this.

We do this in `gatsby-source-drupl` - https://github.com/gatsbyjs/gatsby/blob/747943cdc9bcffcf21df5c6ac83dd2cc9df16713/packages/gatsby-source-drupal/src/gatsby-node.js#L5

Reply:
It should be available in Node 6+ (probably earlier too, but right now we care about 6+)

Reply:
@pieh, thanks for the hint. I added the missing import.